pyuploadtool/releases_hosting_provider/github_releases.py

```diff
@@ -55,7 +55,7 @@ def create_release(self, metadata: ReleaseMetadata, artifacts):
             self.logger.warning("push to default branch, creating continuous release")
 
             # not using "latest", as this value is reserved by GitHub
-            metadata.tag = "continuous"
+            metadata.tag = os.getenv("GITHUB_ROLLING_RELEASE_TAG", "continuous")
             metadata.release_name = "Continuous build"
```

Comment:
While we're at it, let's also add an option to overwrite the release name. And as this tool is inspired by and designed as a substitute for the old uploadtool script, let's use `continuous` everywhere rather than `rolling`. I also think "rolling" is more used in the context of operating systems, whereas "continuous" is used in user space software.